containers/ddev-webserver/Dockerfile

```diff
@@ -104,16 +104,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -qq install -o Dpkg::Options::="--for
     libldap-common \
     libpcre3 \
     libpq-dev \
-    libpython3-dev \
     nano \
     ncurses-bin \
     netcat-traditional \
     openssh-client \
     patch \
-    python-is-python3 \
```

Comment:
I wonder if this is a mistake. Maybe this should stay in there.

Reply:
This package adds a symlink, that should be there:
```
$ python -V
bash: python: command not found
$ python3 -V
Python 3.11.2
```